bodhi/server/models.py

```diff
@@ -1149,6 +1153,10 @@ class Update(Base):
 
     user_id = Column(Integer, ForeignKey('users.id'))
 
+    # Greenwave
+    greenwave_policies_satisfied = Column(Boolean, default=True, nullable=False)
```

Comment:
I expect that I'll know about the convenience states by watching fedmsg for messages from taskotron and CentOS CI. So there will probably be a new fedmsg listener that sets those three in the future. This is in the "nice to have" category in my mind, so don't concern yourself about that for now ☺